Bottom: 2c4d3af151a5dfbf6474460299a9869a41d2d991
Top:    8fb0086dfd86dc6908835d636c2e75106b7d1dd9
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-15 12:36:44 +0100

Refresh of silly-space-navigator-interfac

---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index 1d0d76c..d1deef9 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -793,11 +793,7 @@ void ndof_pan_cb (int dx, int dy, int dz, gpointer data)
     ghid_port_ranges_pan (-gport->zoom * 5 * dx,
                           -gport->zoom * 5 * dy, TRUE);
   if (dz)
-#if 1 /* XXX: TEMPORARY HACK */
-    ghid_port_ranges_zoom (gport->zoom * (1.0 - (dz / 100.0)));
-#else
-    zoom_by (1.0 - (dz / 100.0), gport->pcb_x, gport->pcb_y);
-#endif
+    ghid_zoom_view_rel (gport->pcb_x, gport->pcb_y, 1.0 - (dz / 100.0));
 }
 
 void ndof_roll_cb (int dx, int dy, int dz, gpointer data)
